src/models/installation.ts

```diff
@@ -26,6 +26,9 @@ export default class Installation extends Sequelize.Model {
 	static async getForClientKey(
 		clientKey: string
 	): Promise<Installation | null> {
```

Comment:
super /nit   line breaks, 
static async getForClientKey(
		clientKey: string
	): Promise<Installation | null> {
	**vs**
static async getForClientKey( clientKey: string ): Promise<Installation | null> {

I find the latter considerably more readable, but personal pref

Reply:
Completely agree, but currently we keep getting issues with formatting because some of it is based on the IDE and not the editorconfig file.  I need to fix it with the eslint stuff, which I think will be my next task.